mininterface/textual_interface/widgets.py

```diff
@@ -74,27 +83,58 @@ def on_button_pressed(self, event):
         self._link.facet.submit()
 
 
-class SecretInput(MyInput):
-    """A password input widget with toggle functionality."""
+class MySecretInput(Horizontal, Changeable):
+    """A custom widget that combines an input field with a visibility toggle button."""
 
     BINDINGS = [Binding("ctrl+t", "toggle_visibility", "Toggle visibility")]
 
-    def __init__(self, tag: SecretTag, *args, **kwargs):
+    DEFAULT_CSS = """
```

Comment:
Are you able to move the binding ctrl+t somewhere to a method so that we may let the programmer to set it dynamically in the future? (That's another feature, you don't have to elaborate this if you dont want to.)

Reply:
Thanks for the investigation!